guides/v2.1/migration/migration-overview-how.md

```diff
@@ -1,52 +1,39 @@
+  * *Delta* - migrates incremental changes, such as orders and inventory
 
 ## Migration modes (phases)
 
-This guide provides information about migration using the Data Migration Tool that can transfer and adapt data from Magento 1 to Magento 2 stores. The migration in overall is split into three phases (or "modes"):
+This guide provides information about migration using the Data Migration Tool that can transfer and adapt data from Magento 1 to Magento 2 stores. The migration in overall is split into three phases (or modes):
 
 *	Configuration settings
-
 *	Data
-
 *	Delta (changes since the last run)
 
-Each mode is divided into steps, each of which is responsible for transferring particular data (for example, {% glossarytooltip a05c59d3-77b9-47d0-92a1-2cbffe3f8622 %}URL{% endglossarytooltip %} rewrite step, EAV step, settings step, and so on). At the begin of a run, a step checks the table structures of Magento 1 and Magento 2 for consistency. Then the actual data is transferred to Magento 2. In the end, this data is verified.
+Each mode is divided into steps, each of which is responsible for transferring particular data, such as the {% glossarytooltip a05c59d3-77b9-47d0-92a1-2cbffe3f8622 %}URL{% endglossarytooltip %}, the rewrite step, the EAV step, and the settings step. At the beginning of a run, a step checks the table structures of Magento 1 and Magento 2 for consistency. Then the actual data is transferred to Magento 2 and is verified.
 
 This section previews your migration experience by providing a high-level overview of the Data Migration Tool.
 
 ## Conceptual overview
 
-The data migration tool recognizes the differences in database structure between Magento 1.x versions. Most of these database structural differences are declared in *map files*. Each step in the process uses map files to transform data for use in your Magento 2 store.
+The data migration tool recognizes the differences in database structure between Magento 1.x and 2.x versions. Most of these database structural differences are declared in map files. Each step in the process uses map files to transform data for use in your Magento 2 store.
 
-For example, when you transform data from a {{site.data.var.ce}} 1.8.0.0 database to {{site.data.var.ce}} 2.x.x, the map file accounts for the fact that a table was renamed and renames it accordingly in the destination database. If there are no differences in data structure or data format, the Data Migration Tool transfers it as-is to the Magento 2 database, including data from tables created by extensions.
+For example, when you transform data from a {{site.data.var.ce}} 1.8.0.0 database to {{site.data.var.ce}} 2.x.x, the map file accounts for the fact that a table was renamed, and renames it accordingly in the destination database. If there are no differences in data structure or data format, the Data Migration Tool transfers it as-is, including data from tables created by extensions, to the Magento 2 database.
```

Comment:
Don't know if we should capitalize **Data Migration Tool** or not. Not sure if that's an internal tool's title, but not capitalizing is probably cool.

Reply:
I think it is a specific tool title. But that's a good question!